packages/cli/src/commands/generate/sdl/templates/sdl.js.template

```diff
@@ -3,6 +3,14 @@ export const schema = gql`
     ${query}
   }
 
+  <% enums.forEach((enumDef, idx) => { %>
+    enum ${enums[idx].name} {
+      <% enums[idx].values.forEach((enumDefValue, idk) => { %>
+          ${enums[idx].values[idk].name}
+      <% }) %>
+    }
+  <% }) %>
+
```

Comment:
Does this create empty lines between each enum value?

Reply:
Probably, the templates end up looking pretty gross in order to get the output format looking nice. Run it and see what it does!

Reply:
I updated this slightly. And yes, the lines here are what creates the unnecessary spaces.
I decided to have the `sdl.template.js` to be slightly less readable, but by doing that I ensure I do not affect any existing generation files with unnecessary spaces.